test/TestAccountHelper.js

```diff
@@ -0,0 +1,219 @@
+const nearAPI = require('near-api-js');
+
+const constants = require('../constants');
+
+const { SERVER_EVENTS } = constants;
+
+function extractValueFromHash(hash, key) {
+    const value = hash[key];
+    delete hash[key];
+
+    return value;
+}
+
+class TestAccountHelper {
```

Comment:
extracting stuff here might be a good move, but there is way too many huge changes at the same time to review productively